src/commands/Suggestions/suggest.ts

```diff
@@ -24,17 +25,22 @@ export default class extends SkyraCommand {
 	private kChannelPrompt = this.definePrompt('<channel:textChannel>');
 
 	public async run(message: KlasaMessage, [suggestion]: [string]) {
-		// If including a flag of `--global` then revert to legacy behaviour of sending feedback
-		if (Reflect.has(message.flagArgs, 'global')) {
-			return this.client.commands.get('feedback')!.run(message, [suggestion]);
+		// If including a flag of `--global` send suggestions to #feedbacks in Skyra Lounge
+		const globalSuggestion = Reflect.has(message.flagArgs, 'global') && this.client.webhookFeedback;
+
+		const guild = globalSuggestion ? this.client.guilds.get(SUPPORT_SERVER)! : message.guild!;
```

Comment:
This is a bold asumption. One of the reasons why we use a webhook is because they're available anytime regardless of state **and** shard.

If Skyra starts sharding, this code will not longer work and will cause `TypeError`s to be thrown.

Reply:
yeah my idea was that the suggestion would be send by the webhook

Reply:
Added a note about it and since we need to fix more for sharding that'll have to do for now